Sources/SmokeDynamoDB/AWSDynamoDBCompositePrimaryKeyTable+updateItems.swift

```diff
@@ -48,6 +59,14 @@ public extension AWSDynamoDBCompositePrimaryKeyTable {
                                                 existingItem: existing)
         }
         
+        return statement
+    }
+
+    private func entryToBatchStatementRequest<AttributesType, ItemType>(
```

Comment:
We could add the validation to this as well, otherwise the request won't be valid.